pkg/cloud_provider/file/file.go

```diff
@@ -261,6 +267,9 @@ func CompareInstances(a, b *ServiceInstance) error {
 	if a.Network.Name != b.Network.Name {
 		mismatches = append(mismatches, "network name")
 	}
+	if a.KmsKeyName != b.KmsKeyName {
```

Comment:
can you put a comment here why simple string compare works

Reply:
done